ui/src/components/CatalogFilter/CatalogFilter.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { useObserver } from 'mobx-react';
+import Filter from '../Filter/Filter';
```

Comment:
How about `import Filter from '../Filter';` 
Do you know what the "react's way" to  organise pure functional components like `Filter`?

Reply:
You mean following the structure as 

```
/src
  /components
    /App
      /index.tsx
    /CategoryFilter 
      /index.js
    /Filter
      /index.js
```